apps/site/styles/effects.css

```
@@ -10,3 +10,47 @@ h1.special {
     md:leading-[4rem]
     md:-tracking-[0.06rem];
 }
+
+.turtle {
+  @apply animate-surf
+    absolute
+    z-10
+    translate-x-0
+    translate-y-0;
+}
+
+.turtle img {
+  @apply h-auto
+    w-[200px];
+}
+
+.turtle::after {
+  @apply absolute
+    left-[-50%]
+    top-[50%]
+    -z-10
+    block
+    h-[150px]
+    w-[150px]
+    rotate-[-90deg]
+    select-none
+    bg-[url('/static/images/smoke.gif')]
+    opacity-[0.15];
```

same as above, more tailwindy way 😄 

```suggestion
    -left-full
    top-[20%]
    -z-10
    block
    h-36
    w-36
    -rotate-90
    select-none
    bg-[url('/static/images/smoke.gif')]
    opacity-[0.15]
    md:-left-1/2
    md:top-1/2;
```